Automattic/jetpack
# [AI] Display user-friendly errors

Fixes https://github.com/Automattic/jetpack/issues/28571 and https://github.com/Automattic/jetpack/issues/28570

## Proposed changes:
* Displays the error sent by the back-end if needed

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Patch this to your sandbox
* Patch D99484-code
* switch production `bin/jetpack-downloader switch jetpack production`
* Use some personality names for image generation or modify OPENAI_TOKEN 
* Generate soem AI content
* You should see some user-friendly errors
